packages/twenty-front/src/modules/object-record/record-field/ui/types/guards/isFieldImageValue.ts

```
@@ -0,0 +1,23 @@
+import { z } from 'zod';
+
+import { type FieldImageValue } from '@/object-record/record-field/ui/types/FieldMetadata';
+
+import { type FieldImageValue } from '@/object-record/record-field/ui/types/FieldMetadata';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'FieldImageValue'. Consider to delete it or rename it.

'FieldImageValue' is defined here:


(lint/suspicious/noRedeclare)